### PR TITLE
Explain a bit what our tests actually do.

### PR DESCRIPTION
**Pull request by realityking**
@sdepold is that sufficient or would you like more details?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,5 +46,13 @@ from the documentation.
  - All pull requests should be reviewed by somebody else before merging to master. Preferably they are reviewed by @eugenekudashev.
  - To actually make your changes appear on staging/production, follow [this guide](https://github.com/contentful/marketing-website/blob/master/README.md#contribute-to-the-docs)
 
+## Tests
+
+The tests in this repository are using [dedd](https://github.com/apiaryio/dredd) to check that the examples defined in
```
Comment (sdepold) [inline README.md]: typo: 'dedd'

File: README.md [hunk 2]
```diff
@@ -44,5 +46,13 @@ from the documentation.
...
+he docs actually works against the API. Generally this works by taking the URL template and example body, replacing all variables with the
```
Comment (sdepold) [inline README.md]: `he docs` --> `the docs`

Comment (sdepold) [inline README.md]: `actually work` (without the s)

File: README.md [hunk 3]
```diff
@@ -44,5 +46,13 @@ from the documentation.
...
+sample values and comparing the APIs response with the MSON defined for this request. Currently only GET requests are tested.
+
+Note: The tests only affect the blueprint used for the documentation on Apiary. Files in the de `docs/` directory are not tested in any way.
```
Comment (sdepold) [inline README.md]: `de` is a leftover?

**Comment by sdepold** [comment]
rest lgtm

**Comment by realityking** [comment]
That was horrible, sorry :see_no_evil:

**Comment by sdepold** [comment]
lgtm

**Pull request merged**